include/af/graphics.h

```diff
@@ -0,0 +1,88 @@
+/*******************************************************
+ * Copyright (c) 2014, ArrayFire
+ * All rights reserved.
+ *
+ * This file is distributed under 3-clause BSD license.
+ * The complete license agreement can be obtained at:
+ * http://arrayfire.com/licenses/BSD-3-Clause
+ ********************************************************/
+
+#pragma once
+
+#include <af/defines.h>
+#include <af/array.h>
+
+typedef unsigned long long af_window;
+
+typedef struct {
+    int row;
+    int col;
+    const char* title;
+} af_cell;
+
+#ifdef __cplusplus
+#include <utility>
```

Comment:
Yes, what will C-API users do ? without such structure, the API will keep changing if we add more options to each cell rendering in multi-view mode.